src/bp/ui-admin/src/App/Layout.tsx

```diff
@@ -83,7 +76,7 @@ const Footer = props => (
   <footer className="statusBar">
     <div className="statusBar-list">
       <div className="statusBar-item">
-        <strong>v{props.version}</strong>
+        <strong>{props.version}</strong>
```

Comment:
I'd keep the "v" so it matches the studio

Reply:
I'd remove it in the studio. It's useless and ugly.

Reply:
Either one, as long as they match on both UI